app/src/main/java/com/android/solvit/ui/message/MessageScreen.kt

```diff
@@ -29,9 +30,5 @@ fun MessageScreen(navigationActions: NavigationActions) {
         text = "The message screen is under construction",
         style = MaterialTheme.typography.bodyLarge,
         modifier = Modifier.align(Alignment.Center))
-    /*SeekerBottomNavigationMenu(
```

Comment:
Why removing the bottom navigation bar, here we will be stuck in the message screen since we don't have access to other pages buttons.

Reply:
Because it is a screen that must be deleted in the future to replace it with Hassan's when he has finished implementing the messages. I just made this screen so that the bottom bar navigation is filled

Reply:
For this: 

> Feedback for Incorrect Input: In addition to red borders, consider displaying specific error messages below each field to inform users of the exact issue (e.g., "Phone number must be at least 6 digits").

 It is a good idea. I'm going to change this now. Thanks man